src/txgraph.h

```diff
@@ -12,8 +12,7 @@
 #ifndef BITCOIN_TXGRAPH_H
 #define BITCOIN_TXGRAPH_H
 
-/** No connected component within TxGraph is allowed to exceed this number of transactions. */
-static constexpr unsigned CLUSTER_COUNT_LIMIT{64};
+static constexpr unsigned MAX_CLUSTER_COUNT_LIMIT{64};
```

Comment:
In  "txgraph: (feature) make max cluster count configurable and "oversize" state" 15da6fdd19813a08bf9571147f97e22b5390ecf9

Maybe comment why 64 selected and how having limit above this will hurt the node/miner? fwiw we have seen clusters with size more than 64 in the past https://bitcoincore.reviews/25038, but I think it's not a regular occurrence.